addons/crm/static/src/views/crm_kanban/crm_kanban_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-name="crm.KanbanView" t-inherit="web.KanbanView" t-inherit-mode="primary">
```

Comment:
As it's the first time I see this, I have a question. Did you try to let the `t` element empty ? If so, I suppose it was raising an error without the xpath ?